### PR TITLE
en (US) fixes and tweaks

### PR DESCRIPTION
**Pull request by danielweck**
@llemeurfr to be discussed ...

### CODE REVIEW

**Comment by danielweck** [comment]
I think my most controversial change would be the renaming of "book" to "publication" in a number of places. Most other changes are to keep the English verbosity low, the GUI text is easier to parse when it is concise.

**Comment by danielweck** [comment]
I have also renamed "serial annotator mode"  (which doesn't mean much and is hard to translate) and "quick annotator mode" which is equally unclear. IMO

File: src/resources/locales/en.json [hunk 1]
```diff
@@ -488,37 +488,37 @@
...
-            "catalogs": "Borrow books from your local library or discover new books with the OPDS catalogs -- It works like an online catalog and allow you to access a variety of publications from multiple sources.",
+            "catalogs": "Borrow publications from your local library or discover publications via OPDS feeds. Online catalogs allow you to access a variety of publications from multiple sources.",
             "home": "Thorium gets a makeover, let's discover what's new!",
             "readingView1": "Thorium adapts itself to your reading preferences: choose amongst various themes, adjust text parameters or use readaloud while folllowing the text in the screen.",
             "readingView2": "Search, navigate to a precise location, add bookmarks or enjoy the full screen mode for distraction-free reading sessions.",
-            "yourBooks": "Welcome in your personnal bookshelf: filter and easily find a book in the grid view or do a advanced search in the table view."
+            "yourBooks": "Welcome to your personnal bookshelf: filter and easily find a publication in the grid view or do a advanced search in the table view."
```
Comment (llemeurfr) [inline src/resources/locales/en.json:508]: I think there is a typo there on "personnal" -> "personal". I would write "easily filter and find a publication in the grid view, or perform an advanced search in the table view"

File: src/resources/locales/en.json [hunk 2]
```diff
@@ -291,26 +291,26 @@
         "remainingTime": "Loans (remaining days)",
         "renewButton": "Renew",
         "returnButton": "Return",
-        "returnedLcp": "This book can't be read because the LCP licence has been returned.",
-        "revokedLcp": "This book can't be read because the LCP licence has been revoked.",
+        "returnedLcp": "This LCP-protected publication is returned.",
```
Comment (llemeurfr) [inline src/resources/locales/en.json:294]: "is" -> "was", no?

File: src/resources/locales/en.json [hunk 3]
```diff
@@ -271,15 +271,15 @@
             "title": "Duration"
         },
         "expired": "Loan expired",
-        "expiredLcp": "This book can't be read because the LCP licence has expired.",
+        "expiredLcp": "Expired LCP license.",
```
Comment (llemeurfr) [inline src/resources/locales/en.json:274]: better -> "The LCP license has expired". (to be even with returnedLCP and revokedLCP)

File: src/resources/locales/en.json [hunk 4]
```diff
@@ -291,26 +291,26 @@
         "remainingTime": "Loans (remaining days)",
         "renewButton": "Renew",
         "returnButton": "Return",
-        "returnedLcp": "This book can't be read because the LCP licence has been returned.",
-        "revokedLcp": "This book can't be read because the LCP licence has been revoked.",
+        "returnedLcp": "This LCP-protected publication is returned.",
+        "revokedLcp": "This publication's LCP license is revoked.",
```
Comment (llemeurfr) [inline src/resources/locales/en.json:295]: better -> "The LCP license was revoked". (to be even with expiredLCP)

File: src/resources/locales/en.json [hunk 5]
```diff
@@ -188,36 +188,36 @@
...
         "menu": {
-            "aboutBook": "About this book",
-            "addExtract": "Import the extract",
-            "goBuyBook": "Buy this book",
-            "goLoanBook": "Borrow this book",
+            "aboutBook": "About publication",
+            "addExtract": "Import extract",
+            "goBuyBook": "Buy publication",
+            "goLoanBook": "Borrow publication",
             "goRevokeLoanBook": "Revoke loan",
```
Comment (llemeurfr) [inline src/resources/locales/en.json:205]: I'm surprised to find this label: IMO it should not exist

File: src/resources/locales/en.json (hunk 2 above)
Comment (danielweck) [reply src/resources/locales/en.json:294]: it works well as a present "state" (the past refers to the "action"). it's a minor semantic nitpick though, not a deal breaker :)

File: src/resources/locales/en.json (hunk 3 above)
Comment (danielweck) [reply src/resources/locales/en.json]: indeed!

File: src/resources/locales/en.json (hunk 4 above)
Comment (danielweck) [reply src/resources/locales/en.json:295]: in English I feel that expressing the "state" of the publication/license is clearer than expressing the past "action" that may have resulted in the current "state" (users / readers might not necessarily taken the "action" themselves, but at least the "state" is clear and definite).

File: src/resources/locales/en.json (hunk 5 above)
Comment (danielweck) [reply src/resources/locales/en.json:205]: It's an OPDS thing, not LCP...let me check

Comment (danielweck) [reply src/resources/locales/en.json:205]: `http://librarysimplified.org/terms/rel/revoke`

File: src/resources/locales/en.json [hunk 6]
```diff
@@ -259,9 +259,9 @@
             "tracks": "Tracks"
         },
         "author": "Author",
-        "cancelledLcp": "This book can't be read because the LCP licence has been cancelled.",
-        "certificateRevoked": "Certificate is revoked",
-        "certificateSignatureInvalid": "Certificate signature is invalid",
+        "cancelledLcp": "Cancelled LCP license.",
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/locales/en.json]: yes!

**Review by llemeurfr — APPROVED** [review]

**Pull request merged**